bundles/org.openhab.binding.enigma2/src/main/java/org/openhab/binding/enigma2/handler/Enigma2Handler.java

```diff
@@ -47,22 +48,35 @@
 @NonNullByDefault
 public class Enigma2Handler extends BaseThingHandler {
     private final Logger logger = LoggerFactory.getLogger(Enigma2Handler.class);
+    @Nullable
     private Enigma2Configuration config;
+    @Nullable
     private Enigma2Client enigma2Client;
     @Nullable
     private ScheduledFuture<?> refreshJob;
 
     public Enigma2Handler(Thing thing) {
         super(thing);
-        config = getConfigAs(Enigma2Configuration.class);
-        enigma2Client = new Enigma2Client(config.host, config.user, config.password, config.timeout);
     }
 
     @Override
+    @SuppressWarnings("null")
```

Comment:
You seem to add this everywhere, but I don't know what the current coding guidelines are. because I would rather expect to do the proper null checks and not ignore the warnings (unless its a false positive reported by the annotation checker).

Reply:
Fixed, I changed the nullable Fields to Optionals